src/static/css/mobile-carousel.less

```diff
@@ -0,0 +1,113 @@
+/* ==========================================================================
+   cfgov-refresh
+   Overrides for slick carousel on /the-bureau/ and /careers/working-at-cfpb/.
+   ========================================================================== */
+
+.js-mobile-carousel {
```

Comment:
> None of the child blocks need the .js- prefix, since the scripting isn't targeting them specifically.

That's not entirely true. The Slick carousel attaches the `.slick-*` classes to the element with `.js-mobile-carousel` dynamically using JS, but I gotcha all the inside classes shouldn't have `js-` prefixes, such as `js-mobile-carousel_section`.